openmdao/recorders/sqlite_reader.py

```diff
@@ -80,11 +82,16 @@ def __init__(self, filename, pre_load=False):
             The path to the filename containing the recorded data.
         pre_load : bool
             If True, load all the data into memory during initialization.
+        metadata_filename : str
```

Comment:
The metadata_filename argument doesn't seem to be covered by a test (aside from indirect use for the N2).  Is it actually needed? Seems like everything is working with OpenMDAO's guess at a filename.  I suppose the user could rename one and not the other, so then you would need this. Still needs a test to cover this scenario I think.

Reply:
I added a test where `metadata_filename` is used explicitly and expected to work, and another where the filename is bogus so it expects to fail.